test/functorch/test_vmap.py

```diff
@@ -3341,6 +3343,8 @@ def test_vmap_exhaustive(self, device, dtype, op):
         xfail('tensor_split'),
         xfail('to_sparse'),
         xfail('vdot'),
+        xfail('tril'),
```

Comment:
what happens here?

Reply:
added a comment.

This raises `AssertionError: Exception not raised`. I think most likely missing a check in their def of their batching rule.